curricula/admin.py

```diff
@@ -48,19 +48,26 @@ class TopicInline(TabularDynamicInlineAdmin):
     extra = 5
 
 
-class CurriculumAdmin(PageAdmin, VersionAdmin):
+class CurriculumAdmin(PageAdmin, VersionAdmin, FilterableAdmin):
     model = Curriculum
     verbose_name_plural = "Curricula"
     filter_horizontal = ('frameworks',)
     inlines = (TopicInline,)
 
+    def can_access_all(self, request):
+        return request.user.has_perm('curriculum.access_all_curricula')
```

Comment:
should this be

```suggestion
        return request.user.has_perm('curricula.access_all_curricula')
```

?